optimin build failed in clang-12, ubuntu 18.04, add a flag in CMake to fix it.

**Describe the bug**
An error occurs while building `utils/optimin` in clang 12.0.0, ubuntu 18.04 by running `./build_optimin.sh`. 

```
CMakeFiles/optimin.dir/OptiMin.cpp.o:(.data.rel.ro._ZTIN4llvm2cl4listINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEbNS0_6parserIS7_EEEE[_ZTIN4llvm2cl4listINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEbNS0_6parserIS7_EEEE]+0x18): undefined reference to `typeinfo for llvm::cl::Option'
CMakeFiles/optimin.dir/OptiMin.cpp.o:(.data.rel.ro._ZTIN4llvm2cl15OptionValueCopyIbEE[_ZTIN4llvm2cl15OptionValueCopyIbEE]+0x10): undefined reference to `typeinfo for llvm::cl::GenericOptionValue'
CMakeFiles/optimin.dir/OptiMin.cpp.o:(.data.rel.ro._ZTIN4llvm2cl15OptionValueCopyINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEEE[_ZTIN4llvm2cl15OptionValueCopyINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEEE]+0x10): undefined reference to `typeinfo for llvm::cl::GenericOptionValue'
collect2: error: ld returned 1 exit status
src/CMakeFiles/optimin.dir/build.make:103: recipe for target 'src/optimin' failed
make[2]: *** [src/optimin] Error 1
CMakeFiles/Makefile2:393: recipe for target 'src/CMakeFiles/optimin.dir/all' failed
```

**Possible Root Cause**
It seems to be an RTTI issue according to this page [http://klee.github.io/releases/docs/v1.4.0/build-llvm34/](klee-rtti).

**How To Fix It**
By adding a CXXFLAG in `utils/optimin/src/CMakeLists.txt`, add `set(CMAKE_CXX_FLAGS "-fno-rtti")` after `set_executable`. Please don't add the flag in `utils/optimin/CMakeLists.txt` since this flag conflict with `dyncast` in `EvalMaxSAT`. At least it works for me.

```
--- a/utils/optimin/src/CMakeLists.txt
+++ b/utils/optimin/src/CMakeLists.txt
@@ -1,4 +1,5 @@
 add_executable(optimin OptiMin.cpp)
+set(CMAKE_CXX_FLAGS "-fno-rtti")
```

--- a/utils/optimin/src/CMakeLists.txt
+++ b/utils/optimin/src/CMakeLists.txt
@@ -1,4 +1,5 @@
 add_executable(optimin OptiMin.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
 
 foreach(LIB MaLib EvalMaxSAT glucose)
     target_include_directories(optimin PRIVATE

